resources/views/settings/index.blade.php

```diff
@@ -366,7 +366,7 @@
                         <input type="text" class="form-control" name="key" placeholder="{{ __('voyager::voyager.settings.help_key') }}" required="required">
                     </div>
                     <div class="col-md-3">
-                        <label for="asdf">{{ __('voyager::voyager.generic.type') }}</label>
+                        <label for="type">{{ __('voyager::voyager.generic.type') }}</label>
```

Comment:
Shouldn't the <select> have the id="type" property? As far as I can see without context, the "for" attribute has no meaning

Reply:
Yes, it could be removed completely, but for readability I would just keep it with a _real_ name